src/scenes/lobbyscreen.ts

```diff
@@ -103,14 +96,19 @@ export class LobbyScreen extends Phaser.Scene {
         navigator.clipboard.writeText(link);
     }
 
-    private onStartButton(): void {
-        //server distribute roles
-        log("Start");
+    private async onStartButton(): Promise<void> {
+        const all: Nullable<Player[]> = PlayerManager.getAll();
+        if (!all || all.length < 3) {
+            //return;
```

Comment:
This probably not implemented yet for debugging

Reply:
Jaein, it is implemented but i disabled it so that we can test it with the need to simulate 2 more players.

Reply:
Makes sense!